### PR TITLE
fix(authn): fix user search API

### PR DESCRIPTION
**Pull request by savonarola**

### CODE REVIEW

File: apps/emqx_authn/i18n/emqx_authn_api_i18n.conf [hunk 1]
```diff
@@ -204,24 +204,13 @@ emqx_authn_api {
     }
   }
 
-  like_username {
+  like_user_id {
     desc {
-      en: """Fuzzy search username."""
+      en: """Fuzzy search user_id (username or clientid)."""
       zh: """使用用户名模糊查询。"""
```
Comment (savonarola) [inline apps/emqx_authn/i18n/emqx_authn_api_i18n.conf:210]: We should provide correct translation here.

File: apps/emqx_authn/i18n/emqx_authn_api_i18n.conf [hunk 2]
```diff
@@ -204,24 +204,13 @@ emqx_authn_api {
...
-  }
-
-  like_clientid {
-    desc {
-      en: """Fuzzy search clientid."""
-      zh: """使用客户端标识符模糊查询。"""
-    }
-    label {
-      en: """like_clientid"""
+      en: """like_user_id"""
```
Comment (savonarola) [inline apps/emqx_authn/i18n/emqx_authn_api_i18n.conf:213]: We should provide correct translation here.

File: apps/emqx_gateway/i18n/emqx_gateway_api_authn_i18n.conf [hunk 3]
```diff
@@ -77,20 +77,13 @@ emqx_gateway_api_authn {
         }
     }
 
-    like_username {
+    like_user_id {
         desc {
-            en: """Fuzzy search by username"""
+            en: """Fuzzy search by user_id (username or clientid)"""
```
Comment (savonarola) [inline apps/emqx_gateway/i18n/emqx_gateway_api_authn_i18n.conf:82]: We should provide correct translation here.

**Review by thalesmg — APPROVED** [review]

File: apps/emqx_authn/i18n/emqx_authn_api_i18n.conf (hunk 1 above)
Comment (HJianBo) [inline apps/emqx_authn/i18n/emqx_authn_api_i18n.conf:210]:
```suggestion
      zh: """使用用户 ID （username 或 clientid）模糊查询。"""
```

File: apps/emqx_authn/i18n/emqx_authn_api_i18n.conf [hunk 4]
```diff
@@ -204,24 +204,13 @@ emqx_authn_api {
...
       zh: """模糊用户名"""
```
Comment (HJianBo) [inline apps/emqx_authn/i18n/emqx_authn_api_i18n.conf:214]: I don't known how to say in Chinese🙃 Just using `like_user_id` ?
```suggestion
      zh: """like_user_id"""
```

File: apps/emqx_gateway/i18n/emqx_gateway_api_authn_i18n.conf [hunk 5]
```diff
@@ -77,20 +77,13 @@ emqx_gateway_api_authn {
...
             zh: """Username 模糊搜索"""
```
Comment (HJianBo) [inline apps/emqx_gateway/i18n/emqx_gateway_api_authn_i18n.conf:83]:
```suggestion
            zh: """用户 ID （username 或 clientid）模糊搜索"""
```

**Review by HJianBo — APPROVED** [review]

**Pull request merged**